src/graphics/program-lib/chunks/combineDiffuseSpecularNoReflSeparateAmbient.frag

```diff
@@ -2,4 +2,3 @@ uniform vec3 material_ambient;
 vec3 combineColor() {
     return (dDiffuseLight - light_globalAmbient) * dAlbedo + dSpecularLight * dSpecularity + material_ambient * light_globalAmbient;
 }
```

Comment:
Just revert this file. You actually want the newline at the end because it formats constructed shaders better anyway.

Reply:
reverted the file